packages/relay-kit/src/packs/safe-4337/Safe4337Pack.ts

```diff
@@ -304,20 +296,14 @@ export class Safe4337Pack extends RelayKitBasePack<{
       })
     }
 
-    const usePaymaster = userOperation.paymasterAndData !== '0x'
-    const hasSafeDeployment = userOperation.initCode !== '0x'
-
-    // adjustment only needed for deployments without paymaster
-    if (!usePaymaster && hasSafeDeployment) {
-      const adjustEstimationData = await feeEstimator?.adjustEstimation?.({
-        bundlerUrl: this.#BUNDLER_URL,
-        entryPoint: this.#ENTRYPOINT_ADDRESS,
-        userOperation: safeOperation.toUserOperation()
-      })
+    const adjustEstimationData = await feeEstimator?.adjustEstimation?.({
```

Comment:
This adjustment helps to avoid [the AA40 over `verificationGasLimit` error](https://docs.pimlico.io/bundler/reference/entrypoint-errors/aa40#aa40-over-verificationgaslimit).

Now we always do the adjustment, no matter if there's a paymaster or not.